### PR TITLE
fix(packages): fix eslint, husky, semantic commits

### PR DESCRIPTION
**Pull request by davidholyko**
Summary:

* Moves husky, semantic commits to dependencies
* Fixes eslint versioning

### CODE REVIEW

**Comment by PatrickDohn** [comment]
Looks good to me

File: .eslintrc.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "env": {
     "browser": true,
-    "es2021": true,
+    "es2020": true,
```
Comment (davidholyko) [inline .eslintrc.json:4]: why is this needed?
